bundle/language_en.properties

```diff
@@ -387,27 +448,22 @@ angal.common.codestar
 angal.common.date                                                                = Date
 angal.common.datefrom                                                            = Date From:
 angal.common.datem                                                               = DATE
-angal.common.dateto                                                              = Date To:   
-angal.common.delete                                                              = Delete
-angal.common.deletem                                                             = DELETE
+
 angal.common.description                                                         = Description
 angal.common.descriptionm                                                        = DESCRIPTION
 angal.common.descriptionstar                                                     = Description *
```

Comment:
just a note: also for `angal.common.<key>star` elements we could find a way to add the star programmatically (not now, but in future!!!)

Reply:
Please read the note at line 1757; those keys are hardcoded in the sql files and I decided not to touch them at this time.

Reply:
I think we can wait for this change

Reply:
Agreed; not in scope for these changes.